Source/com/drew/imaging/ImageMetadataReader.java

```diff
@@ -132,17 +132,17 @@ public static Metadata readMetadata(@NotNull final InputStream inputStream, fina
     @NotNull
     public static Metadata readMetadata(@NotNull final InputStream inputStream, final long streamLength, @Nullable String filePath) throws ImageProcessingException, IOException
     {
-        BufferedInputStream bufferedInputStream = inputStream instanceof BufferedInputStream
-            ? (BufferedInputStream)inputStream
-            : new BufferedInputStream(inputStream);
+        try (BufferedInputStream bufferedInputStream = inputStream instanceof BufferedInputStream
+                ? (BufferedInputStream)inputStream : new BufferedInputStream(inputStream)) {
```

Comment:
I don't think this method should be taking ownership of the `inputStream` parameter. The caller may wish to use it after calling this method. My sense is that the changes to this file should be undone.